base/docs/helpdb/Dates.jl

```diff
@@ -82,8 +128,9 @@ Dates.Date(dt::DateTime)
 """
     Date(dt::AbstractString, format::AbstractString; locale="english") -> Date
 
-Construct a `Date` type by parsing a `dt` date string following the pattern given in the
-`format` string. Follows the same conventions as `DateTime` above.
+Construct a `Date` object by parsing a `dt` date string following the pattern given in the
+`format` string. Follows the same conventions as
+`DateTime(::AbstractString, ::AbstractString)`.
```

Comment:
Both of those suggestions don't work when looking up method help:
```julia
help?> DateTime(::AbstractString, ::AbstractString)
  DateTime(dt::AbstractString, format::AbstractString; locale="english") -> DateTime

  Construct a DateTime by parsing the dt date string following the pattern given in the format string. The following character codes can be used to construct the format string:

Code     Matches   Comment                                                   
–––––––– ––––––––– ––––––––––––––––––––––––––––––––––––––––––––––––––––––––––
y        1996, 96  Returns year of 1996, 0096                                
m        1, 01     Matches 1 or 2-digit months                               
u        Jan       Matches abbreviated months according to the locale keyword
U        January   Matches full month names according to the locale keyword  
d        1, 01     Matches 1 or 2-digit days                                 
H        00        Matches hours                                             
M        00        Matches minutes                                           
S        00        Matches seconds                                           
s        .500      Matches milliseconds                                      
e        Mon, Tues Matches abbreviated days of the week                      
E        Monday    Matches full name days of the week                        
yyyymmdd 19960101  Matches fixed-width year, month, and day                  

  All characters not listed above are treated as delimiters between date and time slots. So a dt string of "1996-01-15T00:00:00.0" would have a format string like "y-m-dTH:M:S.s".

help?> DateTime(dt, format)
ERROR: UndefVarError: dt not defined
 in eval(::Module, ::Any) at ./boot.jl:267

help?> DateTime(dt::AbstractString, format::AbstractString)
ERROR: UndefVarError: dt not defined
 in eval(::Module, ::Any) at ./boot.jl:267
```
Should I still update it?